### ISSUE TITLE
Purchasing shop item requires multiple inventory slots be open, even though the item stacks

### ISSUE THREAD

**Issue by minecrafter** [Wynncraft/Issues#4013]
As an example, try purchasing wheat from any Wheat Merchant while you have a full inventory with at least one chat.

This is confusing and flies in the face of logic.

**Comment by Dar-krusos** [comment]
Yes, this is the same with all merchants and items and needs to be fixed.

**Comment by TrashcanMan** [comment]
I think it's the way the merchants are coded, they just search for empty slots in your inventory. I suppose they could be re-coded to search for both empty slots and slots that the item can be stacked into, but that's more resource-intensive.

**Comment by KeepoPrideHD** [comment]
I bet coders are working on it, but let me tell you an easy way to get over it.
Throw out anything of your inventory for less than 30 seconds, in that time only you will be able to pick that up. You're welcome.

**Comment by Moe-Ronickah** [comment]
Not fixed, yet.
